### ISSUE TITLE
How to train the model to get the best performance with limited resources

### ISSUE THREAD

**Issue by zhangtingyu11** [blakechen97/SASA#16]
I only have a RTX 4090 GPU, and I only can set the total batch_size to 8. However, no matter what learning rate I choose, I can not train the model to reproduce the 85.91 mAP(I only get 85.5 mAP or even worse). Is there any methods to further increase 
the performance? And I found that training process is not stable(the best result may be the 75 repoch not the last epoch, and the best result varies among the different training processes(e.g. best mAP between 85.2 and 85.5 for the same hyper-parameters)). Is your training process the same with mine? Hoping for your reply!
